samples/Mvc.Client/Startup.cs

```diff
@@ -9,17 +9,9 @@
 
 namespace Mvc.Client;
 
-public class Startup
+public class Startup(IConfiguration configuration, IHostEnvironment hostingEnvironment)
 {
-    public Startup(IConfiguration configuration, IHostEnvironment hostingEnvironment)
-    {
-        Configuration = configuration;
-        HostingEnvironment = hostingEnvironment;
-    }
-
-    public IConfiguration Configuration { get; }
-
-    private IHostEnvironment HostingEnvironment { get; }
+    public IConfiguration Configuration { get; } = configuration;
```

Comment:
Should this property be removed too for consistency with the `IHostEnvironment HostingEnvironment` property?

Reply:
This one was left because it's public and the other was removed because it was private. I can make them consistent with each other if you like (either both private or both public).